components/Chips/examples/ChipsInputExampleViewController.m

```diff
@@ -80,7 +96,11 @@ - (void)chipField:(MDCChipField *)chipField didAddChip:(MDCChipView *)chip {
   } else {
     [chip applyThemeWithScheme:self.containerScheme];
   }
+  chip.mdc_adjustsFontForContentSizeCategory = YES;
   [chip sizeToFit];
+
+  _chipField.chipHeight = MAX(_chipField.chipHeight, chip.frame.size.height);
```

Comment:
I need to change this to an if not zero then set; the max will give weird behavior for smaller fonts.